main/src/cgeo/geocaching/maps/CGeoMap.java

```diff
@@ -330,6 +331,7 @@ private void showProgress(final boolean show) {
                 progress.setVisibility(visibility);
             }
             map.activity.setProgressBarIndeterminateVisibility(show);
```

Comment:
If I get it right, we can remove the activity.setProgress... calls everywhere, since they are not doing anything anymore?

Reply:
Will update accordingly. There is probably more code specific to the old actionbar progress spinner. Do you know where the layout of the actionbar is defined?

Reply:
For the setProgressBarIndeterminateVisibility deprecation, also 
the initialization with Window.FEATURE_INDETERMINATE_PROGRESS
is deprecated and needs to be removed.

See
https://developer.android.com/reference/android/app/Activity#setProgressBarIndeterminateVisibility(boolean)
https://developer.android.com/reference/android/view/Window.html#FEATURE_INDETERMINATE_PROGRESS

Reply:
done